proto/options.proto

```diff
@@ -312,6 +312,21 @@ message Costing {
     uint32 axle_count = 81;
     float use_lit = 82;
     bool disable_hierarchy_pruning = 83;
+    oneof has_exclude_bridges {
```

Comment:
why `oneof`?`isn't this defaulting to `false`?

Reply:
I am wondering then why is it used for has_exclude_cash_only_tolls this also defaults to false!